doc/HOWTO_feature_flags.md

```diff
@@ -2,59 +2,116 @@
 
 ## Style/Display
 
-* `use_manage` Use "Use" for the link to the home page and place the Manage Facilities link on the left, or use "Home" for the link to the home page and place the Manage Facilities link to the right
-* `facility_banner_notice` Allow setting a highly visible banner that will display when a user visits the facility homepage
+* `azlist` Display one long list of facilities, or an alphabetized collection (only facilities that start with A, and ...a top nav to access facilities that start with other letters)
 * `daily_view` Display a public daily view for facility's reservation schedule
 * `equipment_list` Display a list of instruments on facility home page
+* `facility_banner_notice` Allow setting a highly visible banner that will display when a user visits the facility homepage
+* `facility_tile_list_admin` Adds the ability to add and remove images from facilities.  These images will only be displayed on the home page if `facility_tile_list` is set to true.
+* `facility_tile_list` Display a grid view of the facility list on the home page, this will include images of the facilities, if the given facility has an image. This does not enable the ability to add or remove images from facilities -- this works independently of `facility_tile_list_admin`, though you may want to use these flags together.
 * `limit_short_description` Limit the short description section for facilities (300 characters)
 * `product_list_columns` Display product lists in columns or one long list
-* `azlist` Display one long list of facilities, or an alphabetized collection (only facilities that start with A, and ...a top nav to access facilities that start with other letters)
-* `facility_tile_list` Display a grid view of the facility list on the home page, this will include images of the facilities, if the given facility has an image. This does not enable the ability to add or remove images from facilities -- this works independently of `facility_tile_list_admin`, though you may want to use these flags together.
-* `facility_tile_list_admin` Adds the ability to add and remove images from facilities.  These images will only be displayed on the home page if `facility_tile_list` is set to true.
+* `use_manage` Use "Use" for the link to the home page and place the Manage Facilities link on the left, or use "Home" for the link to the home page and place the Manage Facilities link to the right
 
-## Onboarding
+## Users and Authentication
 
+* `create_users` Should admins be able to manually add users
 * `devise/lock_strategy`Lock account after 5 failed attempts 
+* `lookup_netids`: If enabled, allow internal and external users to be created. Add additional step to ensure the NetId or email is not already present. Else, navigate to create external user (no internal user creation allowed).
```

Comment:
This _else_ is for when the FF is off? Can we be more explicit? The sentence between the _if_ and _else_ is a bit confusing for me.